### DISCUSSION TITLE
Any thoughts about adding a 'recent posts' section?

### DISCUSSION THREAD

**Discussion by headforthecloud** [Lednerb/bilberry-hugo-theme#508] [Ideas]
Several other themes add a recent posts section - was just wondering if there were any plans to add something similar.

**Comment by igor-baiborodine** [comment]
@headforthecloud, the "Latest Post" section is already in the footer. If you meant something different, please provide an example.

![image](https://user-images.githubusercontent.com/1027701/221189670-bbe2e98c-aee4-4cf0-a9a8-07cad2308935.png)
